Ocelot.sln.DotSettings

```diff
@@ -0,0 +1,2 @@
+﻿<wpf:ResourceDictionary xml:space="preserve" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml" xmlns:s="clr-namespace:System;assembly=mscorlib" xmlns:ss="urn:shemas-jetbrains-com:settings-storage-xaml" xmlns:wpf="http://schemas.microsoft.com/winfx/2006/xaml/presentation">
+	<s:Boolean x:Key="/Default/UserDictionary/Words/=appsettings/@EntryIndexedValue">True</s:Boolean></wpf:ResourceDictionary>
```

Comment:
New line before EOF is expected

Reply:
It shouldn't be part of code, it's resharper related, let's put that in the ignore list

Reply:
😮  Do you use ReSharper to write code for this project?
I **do not** recommend to use JetBrains tools for this project. I believe a developer should use Visual Studio IDE only. The project has standard settings styling, conventions, merge tools etc. for Visual Studio only.
That is why I see so strange diff highlighted by GitHub diff comparer in PRs... Keep in mind, that we have special line-ending settings!

If you use ReSharper IDE, please do not add any ReSharper files to the repo!

Reply:
@raman-m As I wrote once, I'm on several projects, sometimes I forget... That's why I'm happy to work with quality gates.